src/main/scala/pl/multiplex/models/TicketType.java

```diff
@@ -0,0 +1,24 @@
+package pl.multiplex.models;
+
+public enum TicketType {
+
+  ADULT(25d, "zł"),
+  STUDENT(18d, "zł"),
+  CHILD(12.5, "zł");
+
+  private Double price;
```

Comment:
Czy $$ jako double to dobry pomysł?

Reply:
Były to małe wartości i stąd ten double. Jednak wartości te mogłaby sumować jakaś metoda, więc pewnie lepiej byłoby zastosować BigDecimal.